.eslintrc.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
   root: true,
+  parserOptions: {
```

Comment:
All of these config options come from `@ilios/eslint-config-ember-addon` and ndon't need to be specified here.

Reply:
removed.